workitem/link/link_repository.go

```diff
@@ -200,6 +202,11 @@ func (r *GormWorkItemLinkRepository) Load(ctx context.Context, ID uuid.UUID) (*W
 	return &result, nil
 }
 
+// Exists returns true|false whether a work item link exists with a specific identifier
+func (m *GormWorkItemLinkRepository) Exists(ctx context.Context, id string) (bool, error) {
+	return repository.Exists(ctx, m.db, WorkItemLink{}.TableName(), id)
```

Comment:
we miss the goa defer in al lthe package. That has to be added in a different PR